internal/pkg/otel/translate/otelconfig.go

```diff
@@ -421,8 +421,11 @@ func translateEsOutputToExporter(cfg *config.C) (map[string]any, error) {
 	// we also want to use dynamic log ids
 	esConfig["logs_dynamic_id"] = map[string]any{"enabled": true}
 
-	// for compatibility with beats, we want bodymap mapping
-	esConfig["mapping"] = map[string]any{"mode": "bodymap"}
+	// logs failed documents at debug level
+	esConfig["telemetry"] = map[string]any{
+		"log_failed_docs_input": true,
+	}
```

Comment:
I don't mind this change, but it is a change. So you're not just adding a test. I'd edit the PR title and mention this in the description, for future generations.